### PR TITLE
sql-math-functions

### LINKED ISSUE #2215: [Concept Entry] SQL Math Functions
### Reason/inspiration (optional)

We would like a new concept entry for Math Functions in SQL.

The entry should go in a file in docs/content/sql/concepts/math-functions/math-functions.md

The entry should: 

- Explain what math functions are in SQL and what they’re used for, starting with an introductory paragraph (2-3 sentences).
- Links out to entries as needed.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by nikolaspoczekaj**
### Description

This Closes #2215 . Thanks for assigning me. Let me know what you think about it. 
Greetings, Niko.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/sql/concepts/math-functions/math-functions.md
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/math-functions.md:3]:
```suggestion
Description: 'SQL Math Functions are built-in features in SQL, which allow mathematical operations directly in database queries.'
```
We don't use first- or second-person pronouns in Docs.

File: content/sql/concepts/math-functions/math-functions.md
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/math-functions.md:17]:
```suggestion
**SQL Math Functions** are built-in features in SQL, which allow mathematical operations and complex calculations, directly in database queries. This makes it easier and more efficient to work with numerical data in SQL, as there's no need to extract the data and process it separately.
```
No first- or second-person pronouns in Docs.

Also, multiplication, addition, subtraction and division are operators, not functions.

File: content/sql/concepts/math-functions/math-functions.md
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/math-functions.md:48]:
```suggestion
This is a practical example of math functions in SQL.
```
From the [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#writing-entries):
>- Avoid words or phrases that assume the reader’s knowledge such as "easy", "obviously", "simply", "of course", and so on.

File: content/sql/concepts/math-functions/math-functions.md
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/math-functions.md:30]: Multiplication (and addition, subtraction and division) is an operator, we'd like to see an example with an actual function. such as `ABS()`, `FLOOR()` or `CELING()`

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@nikolaspoczekaj I did a first review. Most important is that multiplication is an operator, not a function. You need to change the example to use a SQL math function such as `ABS()` or `ROUND()`.

**Comment by nikolaspoczekaj** [comment]
@SSwiniarski changes done. Thanks for the review.

**Review by SSwiniarski — APPROVED** [review]
@nikolaspoczekaj Looks good for a second review.

File: content/sql/concepts/math-functions/math-functions.md
Comment (yangc95) [inline content/sql/concepts/math-functions/math-functions.md:17]:
```suggestion
In SQL, **Math Functions** are built-in features which perform mathematical operations and complex calculations directly in database queries. This makes it possible to work with numerical data in SQL rather than extracting the data and processing it separately.
```

File: content/sql/concepts/math-functions/math-functions.md
Comment (yangc95) [inline content/sql/concepts/math-functions/math-functions.md:21]:
```suggestion
In this example, the following data is given in a table named `employees`:
```

File: content/sql/concepts/math-functions/math-functions.md
Comment (yangc95) [inline content/sql/concepts/math-functions/math-functions.md:19]: For this concept entry, let's steer away from using examples that would appear as a term entry.

In response to: https://github.com/Codecademy/docs/issues/2218#issuecomment-1559350926

Take this entire example section and move it to that term entry you will create.

File: content/sql/concepts/math-functions/math-functions.md
Comment (yangc95) [inline content/sql/concepts/math-functions/math-functions.md:18]: You can add a sentence here that will preface the table of terms. So in conclusion, this entry will only include this introductory paragraph above. And a line below it (please add a new line in between) that says something like "Below is a list math functions that can be used in various SQL platforms:"

cc: @SSwiniarski would platforms be the term we want to use here to describe the different flavors/implementations of SQL?

File: content/sql/concepts/math-functions/math-functions.md
Comment (yangc95) [reply content/sql/concepts/math-functions/math-functions.md:17]: Can you match the metadata's description to this introduction? TY

**Review by yangc95 — COMMENTED** [review]
Thanks @nikolaspoczekaj for making this concept! A previous PR of yours was accepted first and won't render until this entry is also merged.

I conducted a second review~ Just some things we can omit here but otherwise, great introduction into math functions.

File: content/sql/concepts/math-functions/math-functions.md
Comment (SSwiniarski) [reply content/sql/concepts/math-functions/math-functions.md:18]: @yangc95, I think that works.

**Comment by nikolaspoczekaj** [comment]
@yangc95 @SSwiniarski changes done. i removed the example in will use it in the term entry instead.

File: content/sql/concepts/math-functions/math-functions.md
Comment (yangc95) [inline content/sql/concepts/math-functions/math-functions.md:3]:
```suggestion
Description: 'In SQL, math functions are built-in features which perform mathematical operations and complex calculations directly in database queries.'
```

File: content/sql/concepts/math-functions/math-functions.md
Comment (yangc95) [inline content/sql/concepts/math-functions/math-functions.md:17]:
```suggestion
In SQL, **math functions** are built-in features which perform mathematical operations and complex calculations directly in database queries. This makes it possible to work with numerical data in SQL rather than extracting the data and processing it separately.
```

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
Sweet! @nikolaspoczekaj Here's the live link: https://www.codecademy.com/resources/docs/sql/math-functions

Screenshot 2023-05-23 at 6 53 29 PM

cc: @SSwiniarski

**Pull request merged**
